### PR TITLE
Add 'compact' size option to the progress bar

### LINKED ISSUE #426: Compact Progress Bar in Web Components
As an engineer, I want to have the compact progress bar available in web components, so I can use it in my application.

### PR DESCRIPTION
**Pull request by ryan-henness-trimble**
## Description

Add a new size input property to the Modus Progress Bar which takes either 'default' or 'compact' to change the size of the progress bar.

Closes #426

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update

## How Has This Been Tested?

Manually, unit, and e2e tested.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (including CHANGELOG updates)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by trimbleo** [comment]
Looks good to me.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 ## Dependencies
 
-- Node (v14.18.0)
-- NPM CLI (v6.14.15)
+- Node (v16.14.2)
```
Comment (trimbleo) [inline CONTRIBUTING.md:7]: **Suggestion (non-blocking):**
We can enforce the node / npm engine being used by adding a property to the package.json file.
We have done this on our product repo recently.

**Review by trimbleo — APPROVED** [review]
Looks good to me, happy for you to complete and merge this.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ryan-henness-trimble) [reply CONTRIBUTING.md:7]: Ooo that would be nice. I'll write up an issue for it

**Pull request merged**